tests/v3/test_group.py

```diff
@@ -390,6 +390,64 @@ def test_group_create_array(
     assert np.array_equal(array[:], data)
 
 
+def test_group_array_creation(
+    store: Store,
+    zarr_format: ZarrFormat,
+):
+    group = Group.create(store, zarr_format=zarr_format)
+    shape = (10, 10)
+    # TODO: fix tests with path
+    empty_array = group.empty(name="empty", shape=shape)
+    assert isinstance(empty_array, Array)
+    assert empty_array.fill_value == 0
+    assert empty_array.shape == shape
+
+    empty_like_array = group.empty_like(name="empty_like", prototype=empty_array)
+    assert isinstance(empty_like_array, Array)
+    assert empty_like_array.fill_value == 0
+    assert empty_like_array.shape == shape
```

Comment:
Let's add some test coverage around the store. The top level API has som "auto" store discovery and since we're passing the `path` (type: `StorePath`) instead of the `store` parameter, I'm concerned that we aren't using the store we think we are.

Not 100% sure this is what you want but something like this:

```suggestion
    assert empty_like_array.store is store
```

Reply:
Instead of passing the store into the `path` (which would set the array's `store_path` to `make_store_path(store, mode=mode) / self.store_path / name`, I passed the store and the path separately; this sets the array's `store_path` to `self.store_path / name`. Let me know if this is not the behavior that you're looking for!

Reply:
Yeah! This looks right. Thanks :)